app/Kconfig

```
@@ -182,6 +182,9 @@ config SM_MQTTC
 config SM_FULL_FOTA
 	bool "Full modem FOTA support"
 
+config SM_FULL_MODEM_DFU
```

Let's rename this so the prefix is the feature:
```suggestion
config SM_DFU_MODEM_FULL
```

I know we are not consistent with this due to legacy reasons.